core/templates/dev/head/App.ts

```diff
@@ -110,6 +133,19 @@ oppia.constant('LOGOUT_URL', '/logout');
 // page visited.
 oppia.constant('ENABLE_PROMO_BAR', true);
 
+// TODO(vojtechjelinek): Move these to separate file later, after we establish
+// process to follow for Angular constants (#6731).
```

Comment:
In my plan for Angular upgrade, I already plan to do this, i.e. seperate the constants into a different file according to the module it belongs.

Reply:
Ok, I thought so, I will still keep the TODO here and you can remove it later.